### PR TITLE
[ZT] Access SCIM: Azure AD and Okta

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

**Comment by ranbel** [comment]
Previews:
- [Azure](https://ranbel-access-azure-ad.cloudflare-docs-7ou.pages.dev/cloudflare-one/identity/idp-integration/azuread/)
- [Okta](https://ranbel-access-azure-ad.cloudflare-docs-7ou.pages.dev/cloudflare-one/identity/idp-integration/okta/)

File: content/cloudflare-one/identity/idp-integration/azuread.md [hunk 1]
```diff
@@ -6,99 +6,169 @@ weight: 4
...
+You can create Access and Gateway policies for groups that are not synchronized with SCIM. Azure AD exposes directory groups in a format that consists of random strings, the `Object Id`, that is distinct from the `Name`.
+
+1. Make sure you enable **Support groups** as you set up Azure AD on your Zero Trust dashboard.
 
 2. On your Azure dashboard, note the `Object Id` for the Azure group. In the example below, the group named Admins has an ID of `61503835-b6fe-4630-af88-de551dd59a2`.
 
-![Viewing the Azure group IDs on the Azure dashboard](/cloudflare-one/static/documentation/identity/azure/object-id.png)
+    ![Viewing the Azure group ID on the Azure dashboard](/cloudflare-one/static/documentation/identity/azure/object-id.png)
+
+3. When you create an Access or Gateway policy for an Azure group, you will be prompted to enter the **Azure group ID**. Enter the `Object Id` for the Azure group.
```
Comment (ranbel) [inline content/cloudflare-one/identity/idp-integration/azuread.md]:
```suggestion
3. If building an Access policy, choose the _Azure Groups_ selector. If building a Gateway policy, choose the _User Group IDs_ selector.

 4. In the **Value** field, enter the `Object Id` for the Azure group.
```

File: content/cloudflare-one/identity/idp-integration/azuread.md [hunk 2]
```diff
@@ -6,99 +6,173 @@ weight: 4
...
 
-15. On the [Zero Trust dashboard](https://dash.teams.cloudflare.com), navigate to **Settings** > **Authentication**.
+{{<render file="_enable-scim-on-dashboard.md">}}
 
-16. Under **Login methods**, click **Add new**.
+### 2. Configure SCIM in Azure
 
-17. Click **Azure AD**.
+{{<Aside type="note">}}
+Until Microsoft supports out-of-the-box provisioning for Access, SCIM requires a separate enterprise application from the one created during [initial setup](#set-up-azure-ad-as-an-identity-provider).
```
Comment (kennyj42) [inline content/cloudflare-one/identity/idp-integration/azuread.md]: @ranbel I don't think this is necessary or we should reword it. My vote would be to delete or remove the first clause of the sentence. Just call out that we require a second app. Don't want to throw it back onto MSFT for support.

**Review by kennyj42 — APPROVED** [review]
Added a couple small commits and 1 comment. Ready to go once those are resolved.

File: content/cloudflare-one/identity/idp-integration/azuread.md (hunk 2 above)
Comment (ranbel) [inline content/cloudflare-one/identity/idp-integration/azuread.md]:
```suggestion
SCIM requires a separate enterprise application from the one created during [initial setup](#set-up-azure-ad-as-an-identity-provider).
```

File: content/cloudflare-one/identity/idp-integration/azuread.md [hunk 3]
```diff
@@ -6,99 +6,173 @@ weight: 4
...
-![Registering an application in Azure](/cloudflare-one/static/documentation/identity/azure/name-app.png)
+    - `email`
+    - `offline_access`
+    - `openid`
+    - `profile`
+    - `User.Read`
+    - `Directory.Read.All`
+    - `GroupMember.Read.All`
+    
+Note: More narrow permissions may be used, however this is the set of permissions that are tested against regularly.
```
Comment (ranbel) [inline content/cloudflare-one/identity/idp-integration/azuread.md]:
```suggestion
{{<Aside type="note">}}
More narrow permissions may be used, however this is the set of permissions that are tested and supported by Cloudflare.
{{</Aside>}}
```

**Pull request merged**